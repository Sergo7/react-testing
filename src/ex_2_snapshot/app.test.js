import React from 'react';
import { render, screen, fireEvent } from '@testing-library/react';
 
import App from './App';
 
describe('App', () => {
  test('renders App snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
    
  });
});