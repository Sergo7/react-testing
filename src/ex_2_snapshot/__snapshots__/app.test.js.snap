// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Hello
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque aperiam sed temporibus consequuntur at illum sapiente inventore recusandae impedit. Accusantium cum minima laboriosam eos vitae architecto veritatis vel, veniam aut.
    </p>
  </div>
</DocumentFragment>
`;
